static-assets/components/cstudio-contextual-nav/wcm-site-dropdown-mods/wcm-assets-folder.js

```diff
@@ -1598,6 +1600,12 @@ var storage = CStudioAuthoring.Storage;
               });
             }
 
+            //Clipping
+            this.aMenuItems.push({
```

Comment:
Let’s just add a single new menu option called “Edit”. It opens the player just like the regular click of the file. The player will have all the relevant edit buttons (e.g. clip).

Reply:
The edit option is opening the clipDialog